.travis.yml

```diff
@@ -11,44 +15,57 @@ services:
   - docker
   - redis-server
 
-stages:
-  - check
-  - build
-  - test
-  - run
-
+# NOTE: The smoke test itself will fail because the Travis redis instance does
+# not have keyspace notifications enabled. But we're interested in exercising
+# the testground logic, not the test plan itself. So it's fine.
 jobs:
   include:
-    - stage: check
+    - stage: build_check_test_run
```

Comment:
All smashed together?

Reply:
yeah, each steps adds significant latency and there's no clear benefit to separating them because they were executed sequentially anyway